mdx-models/src/main/java/com/mx/path/model/mdx/model/profile/Profile.java

```diff
@@ -2,9 +2,15 @@
 
 import java.time.LocalDate;
 
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
 import com.google.gson.annotations.SerializedName;
+import com.mx.path.core.common.model.Internal;
 import com.mx.path.model.mdx.model.MdxBase;
 
+@Data
+@EqualsAndHashCode(callSuper = true)
 public final class Profile extends MdxBase<Profile> {
   @SerializedName("birth_date_on")
   private LocalDate birthDate;
```

Comment:
`@SerializedName` is necessary for `birthDate` since we are overriding the default serialization which would be `birth_date`. However, the annotations can be removed from `firstName`, `lastName`, and `middleName` since those can all use the default serialization. Can you remove the unnecessary annotations?